.github/workflows/update-release-matrix-to-confluence.yml

```diff
@@ -0,0 +1,121 @@
+name: Update Confluence Table
+
+on:
+  workflow_dispatch:
+    inputs:
+      rhoai-release-version:
+        description: 'RHOAI Release Version'
+        required: true
+      kueue-version:
+        description: 'Kueue Version'
+        required: true
+      codeflare-sdk-version:
+        description: 'CodeFlare SDK Version'
+        required: true
+      codeflare-operator-version:
+        description: 'CodeFlare operator Version'
+        required: true
+      kuberay-version:
+        description: 'Tested KubeRay version'
+        required: true
+      appwrapper-version:
+        description: 'Tested appwrapper version'
+        required: true
```

Comment:
@Srihari1192 I dont see the issue we only ask for 6 params here?

Reply:
nvm you mean in the release wf

Reply:
yes in release workflow